### ISSUE TITLE
How to use for Graph Clustering.

### ISSUE THREAD

**Issue by PeterDeSOM** [Shen-Lab/GraphCL#66]
How can I use GraphCL for fully unsupervised Graph Clustering?

So far, all that I've found the method for Graph Clustering is actually for node clustering or not a fully unsupervised learning method. It means that they eventually need their label to train at the downstream task.

For example, it is;
- pretraining some datasets with the unlabeled using the contrastive learning method. (No output channel or number of classes to be clustered is required)
- then finetuning the dataset with its labels using the supervised learning method.

I'm finding a fully unsupervised learning method for Graph-Level Clustering/Classification that output channel is required. Or, I'd like to know how to employ the typical fully unsupervised learning method to classify lots of graph data that cannot be labeled by human efforts.

Is it possible and could you show me a little direction how to?

Thank you

**Comment by yyou1996** [comment]
Hi @PeterDeSOM,

I guess either pretraining&finetuning or multi-tasking with GraphCL could be the baseline?
